src/cozmo/objects.py

```diff
@@ -286,23 +286,23 @@ def _dispatch_appeared_event(self, changed_fields, image_box):
     def _dispatch_disappeared_event(self):
         self.dispatch_event(EvtObjectDisappeared, obj=self)
 
-    def _handle_available_object(self, available_object):
-        # triggered when engine sends available objects
-        # as a response to a RequestAvailableObjects message
+    def _handle_object_state(self, object_state):
+        # triggered when engine sends objectStates
```

Comment:
Minor, but should be either "ObjectStates" (if the variable name) or "object states". Please reword to:
`# triggered when engine sends an ObjectStates message`